src/test/java/org/pesmypetcare/webservice/thirdpartyservices/adapters/firestore/FirestoreCollectionAdapterTest.java

```diff
@@ -113,34 +139,35 @@ public void getDocumentsWhereFieldEqualToValue() {
             given(collectionReference.whereEqualTo(same(field), same(value))).willReturn(query);
             Query result = collectionAdapter.getDocumentsWhereEqualTo(collectionPath, field, value);
             assertEquals(query, result,
-                "Should return all documents from " + collectionPath + " where " + field + " is equals to " + value);
+                         "Should return all documents from " + collectionPath + " where " + field + " is equals to "
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String " is equals to " appears 4 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5465948)